crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -135,43 +136,44 @@ impl IndexerStore for PgIndexerStore {
 
     fn get_events(
         &self,
-        query: EventQuery,
+        query: EventFilter,
         cursor: Option<EventID>,
         limit: Option<usize>,
         descending_order: bool,
     ) -> Result<EventPage, IndexerError> {
         let mut pg_pool_conn = get_pg_pool_connection(&self.cp)?;
         let mut boxed_query = events::table.into_boxed();
         match query {
-            EventQuery::All => {}
-            EventQuery::Transaction(digest) => {
+            EventFilter::All(..) => {}
```

Comment:
Part of the migration? Or why aren't the inner filters used? Are they checked somewhere else?

Reply:
ya we will implement EventFilter to SQL later in the week, current implementation only process 1 filter, it will process all inner filter as well when it's fully implemented